src/assets/toolkit/styles/components/message.css

```diff
@@ -0,0 +1,38 @@
+/** @define Message */
+
+:root {
+  --Message-padding: var(--space-xs) 2.5em;
+  --Message-background: var(--color-gray);
+  --Message-offline-background: var(--color-orange);
+  --Message-offline-color: var(--color-white);
+  --Message-close-size: 1.4em;
```

Comment:
Also, there is an `.Icon--large` modifier that increases the size by `1.5em`. [More info](http://cloudfour-patterns.netlify.com/patterns/components/icon.html#size-modifiers)